docs/cloud/rahti/rahti-migration.md

```diff
@@ -212,6 +212,16 @@ In the Project details page (`Developer` > `Project`), click `PersistentVolumeCl
 !!! warning "Lazy volume creation"
     The volume will only be created when it is mounted for the first time, this is a change in behavior in `Rahti 2`.
 
+### How to Recreate Pod for Deployment having RWO Volumes
+
+In Rahti 1 the default volume was RWX(read-write-many), so these volumes could be mounted to many pods at same time. But in Rahti 2 volumes are RWO(read-write-once), so these volumes can be mouted to only one pod at a time. 
```

Comment:
Just a small typo. I think "the" is missing:
so these volumes could be mounted to many pods at **THE** same time